src/VisualStudio/Core/Def/Implementation/TableDataSource/Suppression/VisualStudioDiagnosticListTableCommandHandler.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Suppression;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices.Implementation.TableDataSource
+{
+    [Export(typeof(VisualStudioDiagnosticListTableCommandHandler))]
+    internal partial class VisualStudioDiagnosticListTableCommandHandler
+    {
+        private readonly IVisualStudioSuppressionFixService _suppressionFixService;
+        private readonly IVisualStudioDiagnosticListSuppressionStateService _suppressionStateService;
+        
+        [ImportingConstructor]
+        public VisualStudioDiagnosticListTableCommandHandler(
+            IVisualStudioSuppressionFixService suppressionFixService,
```

Comment:
no this is just for the baselining/bulk suppression from error list scenario. It will be used from error list fix and solution explorer -> baseline project/solution.